Packs/CortexXDR/Playbooks/Cortex_XDR_-_Retrieve_File_v2.yml

```diff
@@ -0,0 +1,382 @@
+      '#none#':
+      - "5"
+    scriptarguments:
+      action_id:
+        complex:
+          root: PaloAltoNetworksXDR.RetrievedFiles
+          accessor: action_id
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 565
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: e5d73a46-72d1-4e69-854e-3e28963904ff
+    type: condition
+    task:
+      id: e5d73a46-72d1-4e69-854e-3e28963904ff
+      version: -1
+      name: Is there any file path to retrieve?
+      type: condition
+      iscommand: false
+      description: ''
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "1"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.file_path
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.windows_file_paths
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.linux_file_paths
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.mac_file_paths
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 200
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "5_6_#default#": 0.68
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1110,
+        "width": 635,
+        "x": 10,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: endpoint_ids
+  value: {}
+  required: true
+  description: A comma-separated list of endpoint IDs.
+  playbookInputQuery:
+- key: file_path
+  value: {}
+  required: false
+  description: "A comma-separated list of file paths in any platform. It can be used instead of the macOS/Windows/Linux file paths. \nThe order of the files path list must be parallel to the endpoints list order. Therefore, the first file path in the list is related to the first endpoint ID and so on."
+  playbookInputQuery:
+- key: windows_file_paths
+  value: {}
+  required: false
+  description: |-
+    A comma-separated list of Windows paths.
+    Enter at least one path for either Windows, Linux, or Mac.
+  playbookInputQuery:
+- key: linux_file_paths
+  value: {}
+  required: false
+  description: |-
+    A comma-separated list Linux paths.
+    Enter at least one path for either Windows, Linux, or Mac.
```

Comment:
If this is Linux paths, then shouldn't the description be:
Enter at least one path for Linux.